Gemfile.lock

```diff
@@ -292,7 +290,7 @@ GEM
     geocoder (1.4.4)
     globalid (0.4.0)
       activesupport (>= 4.2.0)
-    graphiql-rails (1.4.4)
+    graphiql-rails (1.4.5)
```

Comment:
I think we should add the line copying `Gemfile.lock` in the generator, so we are sure the version is locked everywhere.

Reply:
Let me check, I tried fixing this "by hand" :/